integration/tests/karpenter/karpenter_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build integration
```

Comment:
:O

Reply:
i don't know what affect this was having, but it felt wrong